ironmon_tracker/Theme.lua

```diff
@@ -618,18 +609,18 @@ function Theme.openSaveCurrentThemeWindow()
 		-- Check a few conditions that would prevent the user from using a particular Theme name
 		if existingPresetIndex == Theme.PresetsIndex.ACTIVE or existingPresetIndex == Theme.PresetsIndex.DEFAULT then
 			-- Don't allow importing "Active Theme (Custom)" or "Default Theme" as that is reserved
-			forms.settext(Theme.Manager.SaveNewWarning, "Cannot use a reserved Theme name")
-			forms.settext(Theme.Manager.SaveNewConfirm, Resources.AllScreens.Save)
+			ExternalUI.BizForms.setText(Theme.Manager.SaveNewWarning, "Cannot use a reserved Theme name")
+			ExternalUI.BizForms.setText(Theme.Manager.SaveNewConfirm, Resources.AllScreens.Save)
 			return
 		elseif themeName:find("%x%x%x%x%x%x") then
 			-- Don't allow six consectuive hexcode characters, as this screws with parsing it later
-			forms.settext(Theme.Manager.SaveNewWarning, "Name cannot have 6 consectuive hexcode characters (0-9A-F)")
-			forms.settext(Theme.Manager.SaveNewConfirm, Resources.AllScreens.Save)
+			ExternalUI.BizForms.setText(Theme.Manager.SaveNewWarning, "Name cannot have 6 consectuive hexcode characters (0-9A-F)")
```

Comment:
typo here: `consectuive` -> `consecutive` (typo was already there but i guess wasn't noticed until now lmao)

Reply:
I'll fix this in dev branch after merger.